enterprise/api_client/enterprise_catalog.py

```diff
@@ -200,14 +200,35 @@ def delete_enterprise_catalog(self, catalog_uuid):
             )
             return {}
 
+    def traverse_get_content_metadata(self, endpoint, query, catalog_uuid):
+        """
+        Helper method to traverse over a paginated response from the enterprise-catalog service's `get_content_metadata`
+        endpoint.
+        """
+        content_metadata = OrderedDict()
+        try:
+            response = endpoint.get(**query)
+            for item in utils.traverse_pagination(response, endpoint):
+                content_id = utils.get_content_metadata_item_id(item)
+                content_metadata[content_id] = item
+        except (SlumberBaseException, ConnectionError, Timeout) as exc:
+            LOGGER.exception(
```

Comment:
incrementally improvable: we can use the existing pattern to log, that uses a nicely extracted out lambda like thingie I recently used in one place, hope to clean that out someday

Reply:
we can ignore this/resolve, since there's no duplicatoin of args issue here